qml/es1_1/es1_1/SwitchButton.qml

```diff
@@ -0,0 +1,47 @@
+import QtQuick
+
+Rectangle {
+    id: button
+    width: 240
+    height: 240
+    radius: 120
+    anchors.centerIn: parent
+    state: "off"
+
+    MouseArea {
+        id: myMouseArea
+        anchors.fill: parent
+        onClicked: parent.state = "on"
```

Comment:
Puoi provare a moficare questa funzione in modo che se lo stao è "on" lo cambia a "off" e viceversa? Così non c'è bisogno di aggiungere il `PropertyChange` che modifica `onClicked`.